src/main/java/step1/Expression.java

```diff
@@ -0,0 +1,26 @@
+package step1;
+
+import java.util.List;
+
+public class Expression {
+    private final List<String> elements;
+
+    public Expression(String input) {
+        if (input == null || input.isBlank()) {
+            throw new IllegalArgumentException("입력값이 비어 있습니다.");
+        }
+        this.elements = List.of(input.split(" "));
+    }
+
+    public int evaluate() {
+        int result = new Operand(elements.get(0)).getValue();
+
+        for (int i = 1; i < elements.size(); i += 2) {
+            Operator operator = Operator.from(elements.get(i));
+            int nextValue = new Operand(elements.get(i + 1)).getValue();
+            result = operator.apply(result, nextValue);
+        }
+
+        return result;
+    }
+}
```

Comment:
```suggestion
public int evaluate() {
    Operand firstOperand = new Operand(elements.get(0));

    return IntStream.range(0, operatorCount())
            .mapToObj(this::parse)
            .reduce(firstOperand, (left, operation) -> operation.apply(left), (a, b) -> a)
            .toInt();
}
```
- 위 코드가 정답이라는 얘기가 아니에요. getter를 지양하고 객체에 메시지를 보내도록 좀 더 리팩토링할 수 있지 않을까요?